packages/mermaid/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mermaid",
-  "version": "10.4.0",
+  "version": "10.5.0-alpha.1",
```

Comment:
Is it worth adding in your PR description that you've published this PR as https://www.npmjs.com/package/mermaid/v/10.5.0-alpha.1 for testing purposes?

Reply:
Done